x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -150,8 +150,10 @@ export const getThresholdAggregationDataProvider = (
   ];
 };
 
-export const isEqlRule = (ecsData: Ecs) =>
-  ecsData.signal?.rule?.type?.length && ecsData.signal?.rule?.type[0] === 'eql';
+export const isEqlRuleWithGroupId = (ecsData: Ecs) =>
+  ecsData.signal?.rule?.type?.length &&
+  ecsData.signal?.rule?.type[0] === 'eql' &&
```

Comment:
nit: Could consolidate to the following using array item access with optional chaining if you'd like:

```suggestion
  ecsData.signal?.rule?.type?.[0] === 'eql' &&
```